### ISSUE TITLE
Service docker must be restarted after boot2docker start

### ISSUE THREAD

**Issue by Fridus** [boot2docker/boot2docker#984]
After each start `boot2docker` I have to restart `docker` to operate.
How it corrected permanently?
I have this problem since the update 1.7.0

```
➜ boot2docker start
Waiting for VM and Docker daemon to start...
.............ooo
Started.
Writing /Users/xxx/.boot2docker/certs/boot2docker-vm/ca.pem
Writing /Users/xxx/.boot2docker/certs/boot2docker-vm/cert.pem
Writing /Users/xxx/.boot2docker/certs/boot2docker-vm/key.pem

To connect the Docker client to the Docker daemon, please set:
    export DOCKER_HOST=tcp://192.168.59.103:2376
    export DOCKER_CERT_PATH=/Users/xxx/.boot2docker/certs/boot2docker-vm
    export DOCKER_TLS_VERIFY=1

➜ docker ps
An error occurred trying to connect: Get https://192.168.59.103:2376/v1.19/containers/json: x509: certificate is valid for 127.0.0.1, 10.0.2.15, not 192.168.59.103
➜ boot2docker ssh 'sudo /etc/init.d/docker restart'
Need TLS certs for boot2docker,127.0.0.1,10.0.2.15,192.168.59.103
-------------------
Generate server cert
/usr/local/bin/generate_cert --host=boot2docker,127.0.0.1,10.0.2.15,192.168.59.103 --ca=/var/lib/boot2docker/tls/ca.pem --ca-key=/var/lib/boot2docker/tls/cakey.pem --cert=/var/lib/boot2docker/tls/server.pem --key=/var/lib/boot2docker/tls/serverkey.pem
2015/07/13 08:59:25 Generating a server cert

➜ docker ps
CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES

➜ boot2docker stop
➜ boot2docker start
Waiting for VM and Docker daemon to start...
................ooo
Started.
Writing /Users/xxx/.boot2docker/certs/boot2docker-vm/ca.pem
Writing /Users/xxx/.boot2docker/certs/boot2docker-vm/cert.pem
Writing /Users/xxx/.boot2docker/certs/boot2docker-vm/key.pem

To connect the Docker client to the Docker daemon, please set:
    export DOCKER_HOST=tcp://192.168.59.103:2376
    export DOCKER_CERT_PATH=/Users/xxx/.boot2docker/certs/boot2docker-vm
    export DOCKER_TLS_VERIFY=1

➜ docker ps
An error occurred trying to connect: Get https://192.168.59.103:2376/v1.19/containers/json: x509: certificate is valid for 127.0.0.1, 10.0.2.15, not 192.168.59.103
```
Boot2Docker-cli version: v1.7.0
Git commit: 7d89508

**Comment by SvenDowideit** [comment]
this problem should be fixed by the changes I made that will be in v1.7.1

**Comment by Fridus** [comment]
@SvenDowideit Today, It works after the update !
```
boot2docker version
Boot2Docker-cli version: v1.7.1
Git commit: 8fdc6f5
```
